client/stylesheets/_toCat.scss

```diff
@@ -4221,16 +4221,15 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileReceivedUl {
-    width: 55%;
-    margin-left: 21% !important;
-    margin-right: 21% !important;
+    width: 97%;
+    margin: 0;
 }
 
 .profileReceivedLi {
-    width: 100%;
+    height: 475px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border, border-radius, height, margin-bottom, padding